packages/utils/retryFn/retryFn.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * This is a function that retries an async function `numRetries` times before throwing an error.
+ *
+ * @param {Function} asyncFn A function that returns a Promise.
+ * @param {string} numRetries Default = 3.
+ */
+export default async function retryFn<T>(
+  asyncFn: () => Promise<T>,
+  numRetries = 3
+): Promise<T> {
+  try {
+    return await asyncFn();
+  } catch (error) {
+    if (numRetries > 1) {
```

Comment:
I think it seems like `numRetries > 0` would be correct. 

Because if the user inputs 1 for numRetries, it seems like the function will execution only once
It's just a single execution, not a retry

Reply:
Good catch!